recipes/certify/all/conanfile.py

```diff
@@ -67,8 +67,7 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "certify")
-        self.cpp_info.set_property("cmake_target_name", "certify")
-        self.cpp_info.components["_certify"].set_property("cmake_target_name", "core")
+        self.cpp_info.set_property("cmake_target_name", "certify::core")
         self.cpp_info.components["_certify"].requires = ["boost::boost", "openssl::openssl"]
 
         self.cpp_info.components["_certify"].names["cmake_find_package"] = "core"
```

Comment:
shouldn't all lines starting with `self.cpp_info.components["_certify"]` be removed ?

Reply:
The workaround must be kept for compatibility with `cmake_find_package/multi `generators. Now those generators do not listen to the `cmake_*` properties.

Reply:
But components still create an unwanted target in CMakeDeps, correct?

Reply:
Yes, as a side effect it creates the `certify::_certify` target, when we just have Conan 2.0 we will be able to remove that target and keep just the root cpp_info